web/config/environment.js

```diff
@@ -66,7 +66,9 @@ module.exports = function (environment) {
       },
     },
 
-    featureFlags: {},
+    featureFlags: {
+      showEmberAnimatedTools: getEnv("SHOW_EMBER_ANIMATED_TOOLS", false),
+    },
```

Comment:
I think this probably fits more as its own config setting instead of a feature flag.

```suggestion
    showEmberAnimatedTools: getEnv("SHOW_EMBER_ANIMATED_TOOLS", false),

    featureFlags: {},
```